fvm/evm/stdlib/contract.cdc

```diff
@@ -4,8 +4,15 @@ import "FlowToken"
 access(all)
 contract EVM {
 
-     access(all)
-     event CadenceOwnedAccountCreated(addressBytes: [UInt8; 20])
+    // Entitlements enabling finer-graned access control on a CadenceOwnedAccount
+    access(all) entitlement Validate
+    access(all) entitlement Withdraw
```

Comment:
I don't know much about entitlements but do we need to also update the validateCOAOwnershipProof to use a Validate entitlement?

Reply:
That isn't needed, and would actually make the function inaccessible: `validateCOAOwnershipProof` is defined on the contract (`EVM`), not on the resource (`CadenceOwnedAccount`)

Reply:
got it.